speaker_verification/model_evaluation.py

```diff
@@ -1,7 +1,10 @@
 import argparse
-from os.path import join, abspath, dirname
 import csv
+import logging
+import os
 import pathlib
+from datetime import date
+from os.path import join, abspath, dirname
```

Comment:
I'd remove `import os` on L4 since you are only using `os.path.dirname` and `os.path.abspath` which is already imported on L7. Could you refactor your usage of these imports to use the L7 imports?

Reply:
Good catch! Will do that.